### ISSUE TITLE
migrations not working with prefixed db

### ISSUE THREAD

**Issue by ariaieboy** [SaliBhdr/typhoon-iran-cities#10]
because of the hardcoded statement in the migrations 

```php
        DB::statement("ALTER TABLE `$this->tableName` comment 'This table is equal to ostan in farsi'");
```
if we use prefix in our tables migrates won't run and throw sql error

**Comment by SaliBhdr** [comment]
> because of the hardcoded statement in the migrations
> 
> ```
>         DB::statement("ALTER TABLE `$this->tableName` comment 'This table is equal to ostan in farsi'");
> ```
> 
> if we use a prefix in our tables migrates won't run and throw sql error

Thanks for the reply I'll check that.

**Comment by SaliBhdr** [comment]
@ariaieboy 

This issue is fixed on version [2.1.3](https://github.com/SaliBhdr/typhoon-iran-cities/releases/tag/2.1.3) on release [58023bb](https://github.com/SaliBhdr/typhoon-iran-cities/commit/58023bbd12fadda49fad8ba51422d307e6e81879)
